app/src/main/java/com/infomaniak/mail/ui/main/newMessage/NewMessageFragment.kt

```diff
@@ -64,7 +69,10 @@ import com.infomaniak.lib.core.R as RCore
 class NewMessageFragment : Fragment() {
 
     private lateinit var binding: FragmentNewMessageBinding
-    private val newMessageActivityArgs by lazy { requireActivity().navArgs<NewMessageActivityArgs>().value }
+    private val newMessageActivityArgs by lazy {
+        // When deeplink it happens that the navigation extras are not yet initialized, so we don't use the `navArgs` here
```

Comment:
```suggestion
        // When opening this fragment via deeplinks, it can happen that the navigation
        // extras aren't yet initialized, so we don't use the `navArgs` here.
```